### PR TITLE
Integrate motion components with Control Safe dialog

### LINKED ISSUE #4005: Add motion domain selector and force toggle to Control Safe dialog
### Story

As a user creating safe transactions, I want to be able to change the domain of the motion and, if I have permission, to force or not the action.

### Description

When the motion & disputes extension is enabled, the control safe dialog should display the motion domain selector and force toggle that we show in any other action dialog, they should be displayed in the preview dialog. This issue is just to integrate those components and make the values available for the motion saga (AKA: Values are available on submit).

### Design

[Figma link](https://www.figma.com/file/LlVmeMQxHVA04evfrgZyuN/Safe-Control?node-id=724%3A15842)

FireShot Capture 731 - Safe Control – Figma - www figma com

### This issue is complete when...

- [x] The force toggle is integrated into the preview transactions dialog.
- [x] The motion domain selector is integrated into the preview transactions dialog.

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
Added force action toggle and motion domain selector.

The motion domain selector is disabled because you are not supposed to create the motion in any other domain that is not the main one.

FireShot Capture 732 - Actions - Colony - wonker - localhost
FireShot Capture 733 - Actions - Colony - wonker - localhost

Resolves #4005

### CODE REVIEW

**Review by willm30 — CHANGES_REQUESTED** [review]
~~I'm noticing the form's valid even if force is toggled off.~~ Sorry, I got confused. The colony had reputation but the user had none so I was expecting the form to be disabled, but looking at other forms this isn't an issue. 

However, should we change the button text to "Create Motion" when it's not being forced? 


Also, do we want to add issues for adding the motion components to add / remove safe forms?

File: src/modules/dashboard/components/Dialogs/ControlSafeDialog/ControlSafeForm.tsx [hunk 1]
```diff
@@ -498,14 +518,15 @@ const ControlSafeForm = ({
           onClick={() =>
             showPreview ? handleSubmit() : handleShowPreview(!showPreview)
           }
-          text={showPreview ? MSG.buttonConfirm : MSG.buttonCreateTransaction}
```
Comment (willm30) [inline src/modules/dashboard/components/Dialogs/ControlSafeDialog/ControlSafeForm.tsx:501]: We could use the existing pattern `{id: button.confirm}` and `{id: button.createMotion}` here?

**Review by willm30 — COMMENTED** [review]
It's showing up for me in the step before instead of on the preview screen:

createMotion

**Comment by ArmandoGraterol** [comment]
@willm30 
> It's showing up for me in the step before instead of on the preview screen:

 Yeah, it's the step before the one that has the "Create transaction" text (which would correspond to a change to "Create motion" when the extension is installed), and the step after its the one with "Confirm". Does that make sense to you?

**Comment by willm30** [comment]
@ArmandoGraterol Shouldn't it be "Create motion" instead of "confirm"? And then when you toggle force it goes back to "confirm"? There's no toggle on the non-preview page so "Create motion" looks a little out of place, at least comparing with other forms, in my opinion.

**Comment by ArmandoGraterol** [comment]
@willm30 
> @ArmandoGraterol Shouldn't it be "Create motion" instead of "confirm"? And then when you toggle force it goes back to "confirm"?

So the way I see it is like this:

The "Confirm" button text still makes sense in the context of having the "Motion & Disputes" extension enabled, you are confirming that you want a motion created for the transactions, therefore it doesn't really need to be replaced. Now if we compare that with "Create transaction", in the context of having the extension enabled, you are actually creating a motion that is going to execute the transactions you added.

> There's no toggle on the non-preview page so "Create motion" looks a little out of place, at least comparing with other forms, in my opinion.

But then again, no other form of us has a preview dialog :P 

I feel like we could argue that the transition from "Create transaction" > "Create motion" also looks awkward in comparison with "Create transaction" > "Confirm" or "Create motion" > "Confirm", or that the place of the toggle and selector is weird because they are not in the dialog with the actual transaction inputs, like our other actions but again no other action has a preview dialog.

Tbh, if it's still confusing I'm good with leaving the texts as they were before as they still make sense ("Create transaction" > "Confirm") and if there's any issue with it surely anyone with Product will point out a better solution (Move the motion components to the first dialog and then change it to "Create motion" perhaps?)

**Review by willm30 — APPROVED** [review]
@ArmandoGraterol Thanks for  clarifying. I agree with you that one set is not obviously more / less correct than another. I would have no issues with your suggestion of reverting to how it was before I raised the initial query 😄

**Review by jakubcolony — APPROVED** [review]
LGTM!

Screenshot 2022-10-20 at 22 07 18

Screenshot 2022-10-20 at 22 07 06

Should I create an issue similar to #3850 for safe control?

**Pull request merged**